spython/tests/test_utils.py

```diff
@@ -77,8 +75,10 @@ def test_check_get_singularity_version(self):
         from spython.utils import get_singularity_version
         version = get_singularity_version()
         self.assertTrue(version != "")
+        oldValue = os.environ['SPYTHON_SINGULARITY_VERSION']
         os.environ['SPYTHON_SINGULARITY_VERSION'] = "3.0"
```

Comment:
There typically isn't a value set here, so this test is going to fail. Why did you add this?

Reply:
To restore a value potentially set by CI/User. I fixed the code so it works with unset env variable (will push with other changes)

Reply:
Great!